content/applications/sales/sales/amazon_connector/setup.rst

```diff
@@ -1,104 +1,151 @@
-==================================
-Configure Amazon Connector in Odoo
-==================================
+==============================
+Amazon Connector configuration
+==============================
 
-Register your Amazon account in Odoo
-====================================
+Odoo allows users to register an Amazon seller account in the database, but the user **must** have
+a paid Amazon Seller account prior to completing the configuration.
+
+To learn how to set up a paid Seller account on Amazon, log in to Amazon, and select *Start a
+Selling Account* from the *Account & Lists* drop-down menu, located in the header. Once that process
+is complete, proceed to follow the instructions below to register and link that Amazon Seller
+account in Odoo.
```

Comment:
Specificity, wording, missing gui/menu labels.

Here's a rewrite with an added **seealso** hyperlink to make it easier for the user to sign up for a Seller account.

```suggestion
Set up a paid Seller account on Amazon by first logging into the Amazon platform, and navigating to
:menuselection:`Account \& Lists --> Start a Selling Account` from the drop-down menu located in
the header section.

Then on the :guilabel:`Sell with Amazon` page, follow the sign-up process and finally proceed to
follow the instructions below to register and link that Amazon Seller account in Odoo.

.. seealso::
   `Sell with Amazon <https://www.amazon.com/b/?node=12766669011>`
```